src/components/query-viewer-element/query-viewer-element.tsx

```diff
@@ -0,0 +1,108 @@
+import { Component, Prop } from "@stencil/core";
+
+@Component({
+  tag: "gx-query-viewer-element",
+  shadow: false
+})
+export class QueryViewerElement {
```

Comment:
Consider initializing some of the properties with default values. Not saying all properties should have a default value, but it's a good practice.